docs/source/cli_help/advance.rst

```diff
@@ -37,9 +37,6 @@ As an example, if ``git machete status --color=never --list-commits`` is as foll
       | |
       | | Add support and sample for machete-post-slide-out hook
       | o-feature/post-slide-out-hook
```

Comment:
```suggestion
        |
        | Add support and sample for machete-post-slide-out hook
        o-feature/post-slide-out-hook
```

Reply:
@amalota resolved but looks unapplied yet 🤔

Reply:
Huh actually:
1. the change is applied in advance.rst
2. the change is **not** reflected in generated_docs.py
3. `tox -e check-py-docs` is failing on my machine for the current commit (`ffc42ce62af46ce58730f216fa09953fe3c84e59`)
4. **still, the CI somehow passed**... which is not expected 🤔

Reply:
Awww I see why, `TRUE` instead of `true` in CI config... lemme comment a suggestion there; once applied, it should fail the build (assuming `generated_docs.py` remain un-regenerated)

Reply:
Failed indeed, gr8: https://app.circleci.com/pipelines/github/VirtusLab/git-machete/2526/workflows/71d89164-a735-4439-9ade-b232838a2db5/jobs/12280

Reply:
(okay resolving)